packages/story-editor/src/components/library/panes/pageTemplates/defaultPageTemplate.js

```diff
@@ -46,13 +45,12 @@ const PageTemplateButton = styled(Button).attrs({ type: BUTTON_TYPES.PLAIN })`
   border-radius: ${({ theme }) => theme.borders.radius.small};
   overflow: hidden;
 
-  &.${ThemeGlobals.FOCUS_VISIBLE_SELECTOR},
-    &[data-focus-visible-added]
-    [role='presentation'] {
+  &:focus-visible,
+  &[data-focus-visible-added] [role='presentation'] {
```

Comment:
Usage of `data-focus-visible-added` should also be replaced now, right?

Reply:
Yes, good catch